aip/0160.md

```diff
@@ -72,8 +82,9 @@ These are used interchangeably.
 
 ### Comparison Operators
 
-Filtering implementations **must** provide the binary comparison operators `=`,
-`!=`, `<`, `>`, `<=`, and `>=`.
+Filtering implementations **should** provide the binary comparison operators
+`=`, `!=`, `<`, `>`, `<=`, and `>=` for string, numeric, timestamp, and
+duration fields (but **should not** provide them for booleans or enums).
```

Comment:
Enums is an interesting one. Some enums are naturally ordered, others aren't. I'm okay with having this here for the moment, but I suspect it's something we may want to tweak over time.